templates/ehub.json

```diff
@@ -194,7 +194,7 @@
                         "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=',variables('webAppStorageAccountName'),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('webAppStorageAccountName')), '2015-06-15').key1)]",
                         "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING": "[concat('DefaultEndpointsProtocol=https;AccountName=',variables('webAppStorageAccountName'),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('webAppStorageAccountName')), '2015-06-15').key1)]",
                         "WEBSITE_CONTENTSHARE": "[concat(toLower(parameters('Application Name')), '9546')]",
-                        "WEBSITE_NODE_DEFAULT_VERSION": "10.14.1",
+                        "WEBSITE_NODE_DEFAULT_VERSION": "12.13.0",
                         "FUNCTIONS_EXTENSION_VERSION": "~2",
```

Comment:
We use Azure functions version 2. Looks like it doesn't support Node 12 and we need to move to version 3.
Coudl you please double-check https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions

Reply:
Done